_posts/0000-01-06-maguileracanon.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Dear @maguileracanon please change the file name to `0000-01-05-maguileracanon.md` and commit.